solution/backend/content_search/views.py

```diff
@@ -95,3 +102,73 @@ def list(self, request):
             return self.get_paginated_response(serializer.data)
         else:
             return Response(serializer.data)
+
+
+class PostContentTextViewset(viewsets.ViewSet):
+    # authentication_classes = [SettingsAuthentication]
+    # permission_classes = [IsAuthenticatedOrReadOnly]
+    @extend_schema(
+        description="Retrieve list sof uploaded files",
+        request=ContentUpdateSerializer,
+        responses={200: ContentUpdateSerializer}
+    )
+    def update(self, request, *args, **kwargs):
+        post_data = request.data
+        id = post_data['id']
+        text = post_data['text']
+        index = ContentIndex.objects.get(uid=id)
+        index.content = text
+        index.save()
+        return Response(data=f'Index was updated for {index.doc_name_string}')
+
+
+class InvokeTextExtractorViewset(viewsets.ViewSet):
+    @extend_schema(
+        description="Post to the lambda function",
+        request=InvokeTextExtractorSerializer,
+        responses={200: InvokeTextExtractorSerializer}
+    )
+    def extract(self, request, *args, **kwargs):
+        post_data = request.data
+        uid = post_data['uid']
+        post_url = post_data['post_url']
+        index = ContentIndex.objects.get(uid=uid)
+        if not index:
+            return Response({'message': "Index does not exist"})
+        json_object = {
+            'id': uid,
+            'uri': index.url,
+            'post_url': post_url,
+        }
+        if index.file:
+            try:
+                json_object['uri'] = index.file.get_key()
+                json_object['backend'] = 's3'
+                json_object["s3"] = {
+                                        "aws_access_key_id": settings.AWS_ACCESS_KEY_ID,
+                                        "aws_secret_access_key": settings.AWS_SECRET_ACCESS_KEY,
+                                        "aws_storage_bucket_name": settings.AWS_STORAGE_BUCKET_NAME
+                                    }
+            except ValueError:
+                json_object['backend'] = 'web'
+
+        if settings.USE_LOCAL_TEXTRACT:
+            json_object['textract'] = {
```

Comment:
This needs to be added to the request so it knows how to connect to textract.  This isnt using the lambda for dev but what is on your machine.